### PR TITLE
Bdd.yml

### PR DESCRIPTION
**Pull request by django9847**
Created BDD.yml file for continuous integration.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Image for a NYU Lab development environment
-FROM rofrano/nyu-devops-base:su23
+FROM rofrano/pipeline-selenium:latest
```
Comment (IrregularHunterX) [inline .devcontainer/Dockerfile:2]: Rebase off mainline? I just merged in the other story you made where included this change

File: features/environment.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ def before_all(context):
     context.wait_seconds = WAIT_SECONDS
     # Select either Chrome or Firefox
     if 'firefox' in DRIVER:
-        context.driver = get_firefox()
+        context.diver = get_firefox()
```
Comment (IrregularHunterX) [inline features/environment.py:18]: For some reason, I can't make a comment on line 9

Change this back to context.driver

Replace line 9 with
DRIVER = getenv('DRIVER', 'chrome').lower()

Try that?

File: features/environment.py [hunk 3]
```diff
@@ -42,5 +42,4 @@ def get_firefox():
     """Creates a headless Firefox driver"""
     options = webdriver.FirefoxOptions()
     options.add_argument("--headless")
-    return webdriver.Firefox(options=options)    
-    
\ No newline at end of file
+    return webdriver.Firefox(options=options)
```
Comment (IrregularHunterX) [inline features/environment.py:45]: What happens if you try adding

options.add_argument("--no-sandbox") in this method?

Comment (django9847) [reply features/environment.py:46]: Didn't work

Comment (IrregularHunterX) [reply features/environment.py:46]: :/

**Review by IrregularHunterX — APPROVED** [review]

**Pull request merged**